services/migrations/20180430093117-create-error-log.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('error_logs', {
```

Comment:
Generally the other tables and keys have all been camelCase rather than snake_case. I suggest renaming to `errorLogs`